.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ commands:
 executors:
   golang-ci:
     docker:
-      - image: okteto/golang-ci:2.3.1
+      - image: okteto/golang-ci:2.3.2
```

Comment:
why have we updated this and the dockerfile? i think is worth to update this as a separete issue Go update